1.0/resources/tools.md

```diff
@@ -0,0 +1,222 @@
+
+[polyclean](https://github.com/PolymerLabs/polyclean) provides basic Gulp plugins for minifying and cleaning JS, CSS, and HTML.
+
+Install:
+
+    npm install -g polyclean
+
+Usage:
+
+    vulcanize --inline-css --inline-scripts index.html | polyclean
+
+Source: [github.com/PolymerLabs/polyclean](https://github.com/PolymerLabs/polyclean)
+
+### <b>polybuild</b> - all-in-one build tool for optimizing apps {#polybuild}
+
+[polybuild](https://github.com/PolymerLabs/polybuild) is an all-in-one build tool that combines vulcanize, crisper, and polyclean. Although less flexible than using the tools individually, polybuild is an easy solution if you want quick defaults.
+
+Install:
+
+    npm install -g polybuild
+
+Usage:
+
+    polybuild index.html --maximum-crush
+
+Source: [github.com/PolymerLabs/polybuild](https://github.com/PolymerLabs/polybuild)
+
+## Serving tools
+
+### <b>polyserve</b> - web server for developing elements {#polyserve}
+
+[polyserve](https://github.com/PolymerLabs/polyserve) is simple web server for serving `bower_components`, locally. It's useful when developing your own elements. 
+
+polyserve serves the component from the current directory as `/components/{element-name}/`, where `element-name` is defined as the name in `bower.json`. All other dependencies are served from `./bower_components/`.
+
+Install:
+
+    npm install -g polyserve
+
+Usage: 
+
+    cd my-element/
+    polyserve -p 8080
+
+Source: [github.com/PolymerLabs/polyserve](https://github.com/PolymerLabs/polyserve)
+
+### <b>polygit</b> - CDN web service for serving components {#polygit}
+
+[polygit](http://polygit.org/) is proxy server for serving components from a CDN. **It is not meant to be used for production apps**, but is very useful when prototyping and sharing jsbins.
+
+Usage:
+
+    <head>
+      <base href="https://polygit.org/components/"> <!-- saves typing! -->
+      <script src="webcomponentsjs/webcomponents-lite.min.js"></script>
+      <link rel="import" href="paper-button/paper-button.html">
+      <link rel="import" href="iron-selector/iron-selector.html">
+    </head>
+
+For more documentation, see [http://polygit.org](http://polygit.org/).
+
+Source: [github.com/PolymerLabs/polygit](https://github.com/PolymerLabs/polygit)
+
+## Development tools {#dev}
+
+### <b>&lt;seed-element></b> - boilerplate starter for creating a resuable element on Github {#seedelement}
+
+See [Create a reusable element](/1.0/docs/start/reusableelements.html).
+
+### <b>Polymer Starter Kit</b> - starter project for building Polymer apps {#psk}
+
+The Polymer Starter Kit is a complete starting point for building Polymer-based applications See [Using Polymer Starter Kit](/1.0/docs/start/psk/set-up.html) for more information on getting started.
+
+Source: [github.com/PolymerElements/polymer-starter-kit](https://github.com/PolymerElements/polymer-starter-kit)
+
+### <b>polylint</b> - lint projects for errors and common mistakes {#polylint}
```

Comment:
Yeah, to me this is more of a development tools conceptually, even if you use it via a build system...

(I'd also be inclined to put polyserve and maybe polygit in as development tools. Especially polyserve, because it's primarily useful for locally testing reusable components.)